src/main/java/com/ss/training/utopia/agent/entity/Booking.java

```diff
@@ -34,6 +41,18 @@
 	@Column
 	private String stripeId;
 
+	@ManyToOne
+	@JoinColumn(name="flightId")
+	private Flight flight;
+
+	@OneToOne
+	@JoinColumn(name="bookerId")
+	private User booker;
+
+	@OneToOne
+	@JoinColumn(name="travelerId")
```

Comment:
There is no column called travelerId outside of the booking table. It maps to userId not travelerId so that I think doesn't work